PUIR_304b/common/cb_types.txt

```diff
@@ -755,7 +755,44 @@ cut_down_to_size = {
 	}
 }
 
+# Cut Down to size (Reperations)
+cut_down_to_size = {

```

Comment:
I think that they will need separate names? I'm happy to keep the disarming CB as `cut_down_to_size`, but perhaps something like `demand_reparations` is a better name?